app/views/admin/meetings/show.html.haml

```diff
@@ -24,16 +23,20 @@
 
     .row
       .medium-2.columns
-        %strong #{@meeting.spaces} spaces
-        %br
-        %br
+        %p
+          %strong #{@meeting.spaces} spaces
 
     .row
       .large-3.small-6.columns#host
         - if @meeting.venue.present?
           %h4 Venue
-          %strong= @meeting.venue.name
-          %br
+          %p
+            %strong= @meeting.venue.name
+          %strong Accessibility info
```

Comment:
@KimberleyCook I added it outside the of statement because on the admin page I want to show a message to update a sponsor with the information if there is no accessibility info yet. So the header would always be there.

Reply:
Yeah I realised I didn't mean to add that comment as the code wrong hint I deleted it 😂